examples/src/examples/loaders/gsplat.example.mjs

```diff
@@ -45,7 +45,7 @@ app.on('destroy', () => {
 });
 
 const assets = {
-    biker: new pc.Asset('gsplat', 'gsplat', { url: `${rootPath}/static/assets/splats/biker.ply` }),
+    biker: new pc.Asset('gsplat', 'gsplat', { url: `${rootPath}/static/assets/splats/shoe.ply` }),
```

Comment:
`biker` -> `shoe`?

Reply:
Or did you forget to set this back to `biker.ply`?

Reply:
oops, no, unintentional thanks!